frontend/src/app/pages/landing-page/landing-page/landing-page.component.html

```diff
@@ -9,12 +9,11 @@ <h1 class="display-3 m-3 text-center">Digitale FüSim MANV</h1>
         <div class="input-group">
             <input
                 [(ngModel)]="exerciseId"
+                (ngModelChange)="joinInputChanged()"
                 [appAutofocus]="true"
                 (keydown.enter)="joinExercise()"
-                [minlength]="6"
-                [maxlength]="8"
                 required
-                appOnlyNumbersValidator
+                appJoinUrlOrIdValidator
```

Comment:
General feedback:

Instead of allowing both, the id or the url, I would just replace the url with the id on paste.
(e.g. https://stackoverflow.com/questions/50138910/angular-on-paste-event-get-content)

Nobody is going to type in the url letter by letter here. Actually, the only occurence where someone pasted the url in here, was during development. None of our customers has (to my knowledge) during our testruns made this mistake, yet.

Also, we only allow ids with the lengths of 6 or 8. So if you are already at it, you could also change this.

Reply:
I've renamed it to `appJoinIdValidator` and only allow 6 or 8 digits now. I've also reintroduced `[minLength]` and `[maxLength]`, since this automatically ignores every input after the 8th char, but pasting longer URLs still works fine (due to handling the paste event).

Does this look good to you or shall we use only one length validation mechanism (i.e., removing `[minLength]` and `[maxLength]`)?

Reply:
Add a comment in the html (above the input) about why you still want the maxLength and minLength validators, and I'm happy with it.